pootle/apps/virtualfolder/receivers.py

```diff
@@ -0,0 +1,13 @@
+
+
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from virtualfolder.models import VirtualFolder
+
+
+@receiver(post_save, sender=VirtualFolder)
+def vfolder_unit_priority_handler(sender, **kwargs):
+
+    for unit in kwargs["instance"].units.all():
```

Comment:
We are not having in account the scenario where a VirtualFolder changes its location, and then suddenly a bunch of units are not included on it.